packages/web-component/cypress/e2e/edit.cy.js

```diff
@@ -0,0 +1,33 @@
+context("The Readalong Component", () => {
+    /**
+     * Wait for the audio and the SMIL to load.
+     */
+    const EXPECTED_LOADING_TIME = 2000; // ms
```

Comment:
Turns out we don't need this at all. I simply removed it in some .cy.js files, and they still work, because I think when you look for something, Cypress already gives it a changes to should up.

Del also has a different (and much more efficient) wait variant that you can see in https://github.com/ReadAlongs/Web-Component/blob/dev.hide-tr-at-load/packages/web-component/cypress/e2e/customizations.cy.js:

```
context('Testing creator enabled settings', () => {
  beforeEach(() => {
    cy.intercept(/\.ras/).as('text')
    cy.intercept(/\.m4a/).as('audio')
  });

  it('testing creator enforced hiding of translation at load time ', function () {

    cy.visit("/ej-fra/index-translated-no-display.html");
    cy.wait('@text')
    cy.wait('@audio')
    cy.readalong().within(() => {
    ...
```

This version waits exactly as long as needed and no more. Our CI runs much faster now in that branch, so we should apply the same idea to new test suites.

Reply:
Well, never mind that claim, apparently we do need to wait. Sigh.